.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,14 +17,23 @@ https://{{head.ref}}--60f9b557105290003b387cd5.chromatic.com
 ## Description
 Closes <ticket>
 
-## Testing done
-
+## QA Checklist
```

Comment:
Is this checklist for the reviewer of the PR or for the person creating the PR? Or maybe both?

Reply:
In my head the list should be used by both parties to ensure everything is being met properly.